include/ezgl/graphics.hpp

```diff
@@ -25,6 +25,8 @@
 #include "ezgl/camera.hpp"
 
 #include <cairo.h>
+#include <cairo-pdf.h>
+#include <cairo-svg.h>
```

Comment:
Do we have to include cairo-pdf.h and cairo-svg.h in graphics.hpp?

Reply:
Good point; won't these be hidden (the print_png function should do all the work, so no need to expose the cairo-pdf header to the application programmer).